app/config.js

```diff
@@ -2,8 +2,7 @@
 
 module.exports = {
   // Service name
-  serviceName: 'Prototype kit',
-
+  serviceName: 'NHS.UK prototype kit documentation',
```

Comment:
Revert back to the serviceName used by the [current website](https://nhsuk-prototype-kit.azurewebsites.net/docs/how-tos) (which this will replace).

```suggestion
  serviceName: 'Prototype kit',
```